UIPS/UIP-0102.md

```diff
@@ -15,6 +15,9 @@ This proposal defines a routing system for Ames and Fine, Urbit's network protoc
 
 Symmetric routing has better properties for NAT traversal than the current asymmetric routing, it promises to be simpler and easier to reason about, it lets stars participate in packet forwarding, and it moves routing responsibility away from publisher ships toward subscribers, increasing scalability.
 
+## Status
+This UIP has been withdrawn.
+
```

Comment:
The status is optional. I wouldn't add this section for documents whose status you aren't otherwise changing.